Makefile

```diff
@@ -746,8 +740,15 @@ bootstrap-device: bootstrap
 	rm -f $(BUILD_STAGE)/.fakeroot_$$(echo $@ | rev | cut -f2- -d"-" | rev)
 	touch $(BUILD_STAGE)/.fakeroot_$$(echo $@ | rev | cut -f2- -d"-" | rev)
 	mkdir -p $(BUILD_DIST)
-%-install:
-	+TO_INSTALL=1 $(MAKE) $(shell echo $@ | $(SED) 's/-install//')-package
+%-install:: export BUILD_DIST=$(BUILD_ROOT)/build_dist/$(MEMO_TARGET)/$(MEMO_CFVER)/stageinstall
```

Comment:
It should not be "stage install" and should instead be named after the `%` package being packed to keep it from breaking if doing multiple -installs at once.